projects/addon-commerce/components/money/pipes/integer-part.pipe.ts

```diff
@@ -0,0 +1,18 @@
+import {Inject, Pipe, PipeTransform} from '@angular/core';
+import {formatNumber, TUI_NUMBER_FORMAT, TuiNumberFormatSettings} from '@taiga-ui/core';
+
+@Pipe({name: 'tuiIntegerPart'})
+export class TuiIntegerPartPipe implements PipeTransform {
+    constructor(
+        @Inject(TUI_NUMBER_FORMAT) private readonly numberFormat: TuiNumberFormatSettings,
+    ) {}
+
+    transform(value: number, precision: number): string {
+        return formatNumber(
```

Comment:
We should probably refactor this method with an `options` object :)

Reply:
Sorry, could you suggest what I should do?

```ts
transform(value: number, { precision, decimalSeparator, thousandSeparator }): string {
```

or you mean

```ts
transform(value: number, precision: number): string {
    const {decimalSeparator, thousandSeparator} = this.numberFormat;

     return formatNumber(
            Math.floor(Math.abs(Number(value.toFixed(precision)))),
            null,
            decimalSeparator,
            thousandSeparator,
        );

```

Reply:
I mean `formatNumber` should be `formatNumber(value: number, option: Partial<AllTheStuff>)`. That's for the future, not for this PR.

Reply:
added todo